tests/foreman/api/test_setting.py

```diff
@@ -0,0 +1,73 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Setting Parameter values
+
+:Requirement: Setting
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import tier1, stubbed
+from robottelo.test import APITestCase
+
+
+class SettingTestCase(APITestCase):
+    """Implements tests for Settings for API"""
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text(self):
+
+        """Updates parameter "login_text" in settings
+
+        :id: 91c5373d-b928-419d-8509-761adf5b94b0
+
+        :steps:
+
+            1. Create login text with any string
```

Comment:
not sure what is meant by that step, but from logic perspective - the whole `:steps:` section should be removed here